docs/resources/config_system.md

```diff
@@ -15,10 +16,12 @@ The following arguments are supported:
 
 * **project_creation_restriction** - (Optional) Who can create projects within Harbor. Can be **"adminonly"** or **"everyone"**
 
-* **robot_token_expiration** - (Optional) The amount of time in days a robot account will expiry. 
+* **robot_token_expiration** - (Optional) The amount of time in days a robot account will expiry.
```

Comment:
`expiry` --> `expire`

Reply:
The spelling of the old docs is not really part of this PR. Open an issue for that :) 

My vscode just removed the trailing space on save. If we want I’ll revert this trailing space and open a separate PR for that change.